packages/atlas/lib/src/atlas.dart

```diff
@@ -66,6 +66,10 @@ class Atlas extends StatelessWidget {
   ///   * [showMyLocation] parameter.
   final bool showMyLocationButton;
 
+  /// If my location is used if there is no motion on the screen.
```

Comment:
This comment could be rewritten more clearly as: 

```
/// This option enables the map to update its camera position so that 
/// the user's current location is always shown in the center of the screen
```